docs/migration/exchange-migration.md

```diff
@@ -1,25 +1,27 @@
 # Exchange migration guide
-Use this guide to rebrand the original Terra chain as Terra Classic and integrate the new Terra chain. 
+
+Use this guide to rebrand the original Terra chain as Terra Classic and integrate the new Terra chain.
 
 ## General info
 
-Per [governance proposal 1623](https://station.terra.money/proposal/1623), a new chain will be created and will assume the Terra name. 
+Per [governance proposal 1623](https://station.terra.money/proposal/1623), a new chain will be created and will assume the Terra name.
+
+1\. The original chain will still function and be re-branded as **Terra Classic**.
 
-1\. The original chain will still function and be re-branded as **Terra Classic**. 
-- The original Cosmos chain will still run, with market swaps (mint/burn function) disabled. 
+- The original Cosmos chain will still run, with market swaps (mint/burn function) disabled.
```

Comment:
- The original chain will have market swaps (minting and burning functionalities) disabled.

We already mention just above that it is still functioning right?